psd-web/config/locales/en.yml

```
@@ -74,6 +74,14 @@ en:
         no_label: "No, this is business as usual"
         hint: "For example, cases relating to hand sanitiser or personal protective equipment (PPE)"
       success: "Coronavirus status updated on %{case_type}"
+    risk_level:
+      show:
+        title_set: "Set case risk level"
+        title_change: "Change case risk level"
```

Suggestion: structure this so that it's a bit clearer that the two titles are alternatives to each other, eg:

```suggestion
      show:
        title:
          set: "Set case risk level"
          change: "Change case risk level"
```